examples/CombinedImuFactorsExample.cpp

```diff
@@ -123,7 +123,7 @@ boost::shared_ptr<PreintegratedCombinedMeasurements::Params> imuParams() {
   // PreintegrationCombinedMeasurements params:
   p->biasAccCovariance = bias_acc_cov;      // acc bias in continuous
   p->biasOmegaCovariance = bias_omega_cov;  // gyro bias in continuous
-  p->biasAccOmegaInt = bias_acc_omega_int;
+  p->biasAccOmegaInit = bias_acc_omega_init;
```

Comment:
Renaming breaks API and I really think it was meant to be "int" for "integration"

Reply:
Nope, we have `integrationCovariance` in `PreintegrationParams.h` for integration. This was a typo that I fixed.